.circleci/config.yml

```diff
@@ -0,0 +1,101 @@
+version: 2.1
+tagged_build_filters: &tagged_build_filters
+  branches:
+    ignore: /.*/
+  tags:
+    only: /v[0-9]+\.[0-9]+\.[0-9]+/
+test_build_filters: &test_build_filters
+  branches:
+    only: /.*/
+  tags:
+    ignore: /v[0-9]+\.[0-9]+\.[0-9]+/
+jobs:
+  test:
+    docker:
+      - image: circleci/ruby:2.7.0
+    steps:
+      - checkout
+      - run: bundle install
+      - run: rubocop -c .rubocop.yml -a
+  build:
+    docker:
+      - image: circleci/ruby:2.7.0
```

Comment:
I can't accept a PR to add CircleCI where the CircleCI build fails... looks like this is not a valid image, 2.7.0 is still in RC state. Let's do the most recent stable, 2.6.5.

Reply:
Well, now it works in `2.6.5` but results the following `RuboCop` error:
```
.rubocop.yml: Metrics/LineLength has the wrong namespace - should be Layout
Error: unrecognized cop Lint/UnusedLocalVariable found in .rubocop.yml

Exited with code exit status 2
```

Investigating...